spec/index.html

```diff
@@ -1627,13 +1701,34 @@ <h3>RDF Term Constructors</h3>
       <tr id="handle-collection2"                     >                                                                                                                                                                           <td><a data-cite="RDF12-CONCEPTS#dfn-iri">          IRI          </a></td><td>For empty lists, the resulting IRI is <code>rdf:nil</code>. Note the rules for <code>collection</code> in the next section.</td></tr>
-      <tr id="handle-quotedTriple"                    ><td style="text-align:left;"            ><a class="type quotedTriple" href="#grammar-production-quotedTriple"                    >quotedTriple                    </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a></td>
+      <tr id="handle-tripleTerm"                      ><td style="text-align:left;"            ><a class="type tripleTerm"   href="#grammar-production-tripleTerm"                      >tripleTerm                      </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-triple-term">  triple term  </a></td>
         <td>
-          The <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>
+          The <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple term</a>
           is composed of the terms constructed from
           the <a href="#grammar-production-subject"><code>subject</code></a>,
           <a href="#grammar-production-predicate"><code>predicate</code></a>, and
-          <a href="#grammar-production-object"><code>object</code></a> productions.
+          <a href="#grammar-production-ttObject"><code>ttObject</code></a> productions.
+        </td>
+      </tr>
+      <tr id="handle-reifiedTriple"                   ><td style="text-align:left;"            ><a class="type reifiedTriple" href="#grammar-production-reifiedTriple"                 >reifiedTriple                   </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a> | <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a></td>
+        <td>
+          The <a class="type reifiedTriple" href="#grammar-production-reifiedTriple">reifiedTriple</a> production
+          defines an RDF term in addition to a <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple term</a>.
+          The term constructed from this production
+          is composed of an identifier from either the <a href="#grammar-production-iri"><code>iri</code></a>
+          or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a> productions,
+          if present, otherwise from a fresh RDF <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>.
```

Comment:
```suggestion
          The term constructed from this production
          is composed of an optional identifier from either the <a href="#grammar-production-iri"><code>iri</code></a>
          or the <a href="#grammar-production-BlankNode"><code>BlankNode</code></a> productions;
          otherwise, from a fresh RDF <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>.
```